examples/ELZ-ReferralComposition-01.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Composition xmlns="http://hl7.org/fhir">
+    <id value="ELZ-ReferralComposition-01"/>
+    <status value="final"/>
```

Comment:
The element `.meta.profile` is missing from the Composition examples.

Reply:
Done!